glossary.md

```diff
@@ -0,0 +1,25 @@
+# Glossary
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating a [container](#container) and launching a process within it.
+
+## Configuration
+
+The [`config.json`](config.md) and [`runtime.json`](runtime-config.md) files in a [bundle](#bundle) which define the intended [container](#container) and container process.
+
+## Container
+
+An environment for executing processes with configurable isolation and resource limitations (namespaces, resource limits, mounts, …).
```

Comment:
On Tue, Dec 22, 2015 at 10:28:22PM -0800, Brandon Philips wrote:
> > +An environment for executing processes with configurable isolation and resource limitations (namespaces, resource limits, mounts, …).
> 
> just say "for example" instead of the parenthetical and ellipses?

In 6bddc28 → 0f9ec22 I've spun the parenthetical out into a new
sentence:

  For example, namespaces, resource limits, and mounts are all part of
  the container environment.

Does that address your concerns?